templates/galaxy_client_app.mako

```diff
@@ -71,6 +71,43 @@ ${ h.dumps( dictionary, indent=( 2 if trans.debug else 0 ) ) }
     </script>
 </%def>
 
+<%def name="config_plausible_analytics(plausible_server, plausible_domain)">
+    <script>
+        console.log("config_plausible_analytics plausible_server:", '${plausible_server}');
+        console.log("config_plausible_analytics plausible_domain:", '${plausible_domain}');
+    </script>
```

Comment:
is this necessary? if not might remove.

Reply:
It's not necessary, just thought I'd keep the same logging as Google Analytics has.